packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -344,7 +342,7 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
     // Compute the Y position of the popup (auto-complete list). Its bottom
     // will be positioned at the top of the text field. Add 1 includes
     // the TextField border.
-    final yCoord =
+    final num yCoord =
```

Comment:
use double instead. Always prefer int or double instead of num

Reply:
done